src/pages/home/report/ReportActionItem.tsx

```diff
@@ -760,75 +786,54 @@ function ReportActionItem(props) {
                 )}
             </Hoverable>
             <View style={styles.reportActionSystemMessageContainer}>
-                <InlineSystemMessage message={props.action.error} />
+                {/* @ts-expect-error TODO check if there is a field on the reportAction object */}
+                <InlineSystemMessage message={action.error} />
```

Comment:
I was not able to check if this field `error` exists on `ReportAction` type so not sure if I should add this to type of this is mistake

Reply:
Maybe ask on slack, or make an artificial error to test this?